api/admin/controller/__init__.py

```diff
@@ -1810,6 +1810,24 @@ def reset(self):
         create_default_lanes(self._db, flask.request.library)
         return Response(unicode(_("Success")), 200)
 
+    def change_order(self):
+        self.require_library_manager(flask.request.library)
+
+        submitted_lanes = json.loads(flask.request.data)
+
+        def update_lane_order(lanes):
+            index = 0
+            for lane_data in lanes:
+                lane_id = lane_data.get("id")
+                lane = self._db.query(Lane).filter(Lane.id==lane_id).one()
+                lane.priority = index
+                index = index + 1
```

Comment:
This is purely stylistic and it's not worth making the change, but I do this with `for index, lane_data in enumerate(lanes)`.